src/js/popup.js

```diff
@@ -51,8 +51,8 @@ function showSignUpPanel() {
 }
 
 
-function choosePanel(numRemaining, panelId, premium){
-  if (premium){
+function choosePanel(numRemaining, panelId, premium, premiumEnabledString){
+  if (premium && premiumEnabledString === "True"){
```

Comment:
Passing in the string value of `settings.PREMIUM_ENABLED`, so the premium panel is set only if both the user data attribute is premium and if the premium features are available on the site.



And if `premiumEnabledString` doesn't return `True`, the add on defaults back to a 'non-premium' view.